src/osd/PG.cc

```diff
@@ -3569,22 +3569,6 @@ void PG::sub_op_scrub_unreserve(OpRequestRef op)
   clear_scrub_reserved();
 }
 
-void PG::sub_op_scrub_stop(OpRequestRef op)
```

Comment:
@liewegas, sorry for the late response. this change was introduced by @jdurgin in 378f84c1a65875d32ac6b8802a245f1d8de06373, but i don't quite understand how we can stop the ongoing scrub by clearing `scrubber.reserved`.